test/wdio/css-variables/cmp.test.tsx

```diff
@@ -0,0 +1,156 @@
+import './variables.css';
+
+import { Fragment, h } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+
+const css = `:root {
+  --global-background: black;
+  --global-color: white;
+  --body-background: grey;
+}
+body {
+  background: var(--body-background);
+  color: var(--global-color);
+}`;
+
+describe('css-variables', () => {
+  beforeEach(() => {
+    render({
+      template: () => (
+        <>
+          <style>{css}</style>
+
+          <css-variables-no-encapsulation></css-variables-no-encapsulation>
```

Comment:
The original test had a second instance of `css-variables-no-encapsulation`. I don't see a reason why we need this, as we don't change anything in `css-variables-no-encapsulation` in the original test (so there's no state change encapsulation to test). LMK if something jumps out at you that I'm missing here